packages/SketchMorph2-Core.package/M2Backend.class/instance/drawCircle.st

```diff
@@ -1,8 +1,9 @@
 tool selection
 drawCircle
-	self currentPen roundNib: self brushSize.
-	self
+	super drawCircle.
+	self context
 		isErasing: false;
 		mode: #pen;
-		currentTool: #drawCircle;
-		switchMode
+		currentTool: #drawCircle.
+	self switchMode.
+	self viewPort drawCircle
```

Comment:
`switchMode` is used for commiting text in the viewPort. It makes more sense in my opinion to first commit a text before switching to another tool, instead of the other way around. Therefore I would leave it as it is.

Reply:
Right, ok.